rollup.config.js

```diff
@@ -104,11 +120,13 @@ export default [
       format: "cjs",
       sourcemap: true,
     },
-    plugins: [babel({ babelHelpers: "bundled" }), shouldMinify && terser()],
-    external: {
-      "single-spa-react": "./single-spa-react.cjs",
```

Comment:
This is the only part i'm unsure about. Looking at https://cdn.jsdelivr.net/npm/single-spa-react@4.4.0/lib/cjs/parcel.cjs, it seems to do `require('single-spa-react')` rather than `require('./single-spa-react.cjs')` which is what I would have expected given this rollup config. In my update, it still is `require('single-spa-react')`, but I'm unsure whether I'm missing/overlooking something about this.